website/docs/docs/build/documentation.md

```diff
@@ -60,13 +56,19 @@ models:
 
 </File>
 
-
 ## Generating project documentation
-You can generate a documentation site for your project (with or without descriptions) using the CLI.
 
-First, run `dbt docs generate` — this command tells dbt to compile relevant information about your dbt project and warehouse into `manifest.json` and `catalog.json` files respectively. To see the documentation for all columns and not just columns described in your project, ensure that you have created the models with `dbt run` beforehand.
+The default documentation experience in dbt Cloud is [dbt Explorer](/docs/collaborate/explore-projects), available on [Team or Enterprise plans](https://www.getdbt.com/pricing/). Use dbt Explorer to view your project's resources (such as models, tests, and metrics), its [metadata](/docs/collaborate/explore-projects#generate-metadata), and their lineage to gain a better understanding of its latest production state.
```

Comment:
@cafzal introducing explorer content here - what do you think?

also moved this from collaborate -> build your dag. however, there is also an option to have a 'doc your project' folder under the 'build' section. and add doc examples for other resources per Dave's suggestion.

Reply:
sounds good!